### DISCUSSION TITLE
E856A0F4-C002-4E98-98F3-12699BCD826F

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#160] [General]
# HD 158259

HD 158259, also known as TOI-1462, is a G-type main sequence star located 88.2 light-years away from Earth. It has six planets (one super Earth and five mini Neptune) with a mass twice to six times that of Earth. The entire system is very compact, with all six planets orbiting very close to the star, and the outermost distance from the star is only 0.38 times that of Mercury to the Sun

https://celestia.mobi/addon?item=E856A0F4-C002-4E98-98F3-12699BCD826F

**Comment by exocritic5457** [comment]
This gives me flashbacks to K2-138

**Comment by TyphoonRagasa** [comment]
The description has issues, please correct it. There are 5 planets here, not 6.

**Reply by Galaxy-Leaper1** [reply]
Don't worry, there will be an update later
